demo/index.php

```diff
@@ -13,16 +13,16 @@
 use Atk4\Ui\Columns;
 use Atk4\Ui\Layout;
 
-$p = ['t' => [
-    ['name' => 'January', 'sales' => 20000, 'purchases' => 10000],
-    ['name' => 'February', 'sales' => 23000, 'purchases' => 12000],
-    ['name' => 'March', 'sales' => 16000, 'purchases' => 11000],
-    ['name' => 'April', 'sales' => 14000, 'purchases' => 13000],
-]];
-$m = new Model(new Array_($p), 't');
+$t = [
+    1 => ['name' => 'January', 'sales' => 20000, 'purchases' => 10000],
```

Comment:
why no `['id' => 1,` here? If it is not needed, no need for it in the following rows.